src/Mod/AddonManager/AddonManager.py

```diff
@@ -428,15 +431,16 @@ def install(self,repos=None):
             # Tab "Macros".
             macro = self.macros[self.dialog.listMacros.currentRow()]
             if utils.install_macro(macro, self.macro_repo_dir):
-                self.dialog.description.setText(translate("AddonsInstaller", "Macro successfully installed. The macro is now available from the Macros dialog."))
+                self.dialog.description.setText(translate("AddonsInstaller",
+                                                          "Macro successfully installed. The macro is "
+                                                          "now available from the Macros dialog."))
             else:
                 self.dialog.description.setText(translate("AddonsInstaller", "Unable to install"))
 
     def show_progress_bar(self, state):
-
         """shows or hides the progress bar"""
 
-        if state == True:
+        if state is True:
```

Comment:
For simple boolean checks, it's more Pythonic to say
```py
if state:
    ...
if not state:
    ...
```

Reply:
changed